### ISSUE TITLE
[10.0][product_by_supplier]can not see product_tmpl_id in form view

### ISSUE THREAD

**Issue by skypc785308** [OCA/purchase-workflow#577]
HI , i have try to  install in clean database
i can see the product in tree view , but in the form view , the value of product is empty
even i can set a value in form for product, then i can see tree show product A, in form show product B
is it a bug?

sorry for my poor English

![image](https://user-images.githubusercontent.com/14845821/44439894-3a192900-a5f8-11e8-9795-d69850b6b51b.png)
![image](https://user-images.githubusercontent.com/14845821/44439912-50bf8000-a5f8-11e8-9488-1c0f170c2f8b.png)

**Comment by skypc785308** [comment]
i fix it
because the field can show only 1 time in form
if there are two same field in form view , it will cause issue
 remove the follow code in views/product_supplierinfo_view.xml
```xml
<record id="view_product_supplierinfo_form1" model="ir.ui.view">
        <field name="name">product.supplierinfo.form1</field>
        <field name="model">product.supplierinfo</field>
        <field name="inherit_id" ref="product.product_supplierinfo_form_view"/>
        <field name="priority">99</field>
        <field name="arch" type="xml">
            <xpath expr="//field[@name='product_name']" position="before">
                <field name="product_tmpl_id" string="Product"
                invisible="context.get('from_product_form')"
                required="not context.get('from_product_form')"/>
            </xpath>
        </field>
    </record>
```
add context to action view  "record id="action_product_supplier_info" model="ir.actions.act_window"
```xml
<field name="context">{'search_default_supplier':1,'visible_product_tmpl_id':0,'group_by':[]}</field>
```

then you can see normal product_tmpl_id  under pricelist
![image](https://user-images.githubusercontent.com/14845821/44440721-fd4f3100-a5fb-11e8-806c-9f65d5351322.png)

**Comment by pedrobaeza** [comment]
Fixed in 1953f50496249f87510cbe2f81c4ee45f59d96c8
